packages/connect-web/src/browserstacktests/README.md

```diff
@@ -0,0 +1,11 @@
+# Connect-Web Browserstack Tests
```

Comment:
Should this directory maybe live alongside `conformance`? It was moved to `src/` just to mimic how `connect-web-test` was structured.